docling/backend/mspowerpoint_backend.py

```diff
@@ -268,9 +270,20 @@ def handle_title(self, shape, parent_slide, slide_ind, doc):
         return
 
     def handle_pictures(self, shape, parent_slide, slide_ind, doc):
+        # Get the image bytes
+        image = shape.image
+        image_bytes = image.blob
+        # Open it with PIL
+        pil_image = Image.open(BytesIO(image_bytes))
+
         # shape has picture
         prov = self.generate_prov(shape, slide_ind, "")
-        doc.add_picture(parent=parent_slide, caption=None, prov=prov)
+        doc.add_picture(
+            parent=parent_slide,
+            image=ImageRef.from_pil(image=pil_image, dpi=72),
```

Comment:
Do we want to hard-code 72 DPI here? I guess we have no better information about the actual DPI?

Reply:
Added another commit, now extracting image DPI from the input file